src/main/java/ui/Ui.java

```diff
@@ -121,7 +122,7 @@ public void printMedicines(ArrayList<Medicine> medicines) {
         String maxQuantityFormat = "%1$-" + maxQuantityWidth + "s | ";
 
         String[] formats = {idFormat, nameFormat, priceFormat, quantityFormat, expiryFormat, descriptionFormat,
-            maxQuantityFormat};
+                            maxQuantityFormat};
```

Comment:
Is this indentation correct? I think indentation for wrapped lines should only be 8 spaces more than parent line.
https://se-education.org/guides/conventions/java/basic.html

Reply:
For now, we will use 12 spaces although the original indentation of 8 is correct by coding standards.